pkg/modules/libreoffice/api/libreoffice_test.go

```diff
@@ -509,6 +509,35 @@ func TestLibreOfficeProcess_pdf(t *testing.T) {
 			start:        true,
 			expectError:  false,
 		},
+		{
+			scenario: "success ExportNotesInMargin",
```

Comment:
Could you also move this before `LosslessImageCompression`?

Reply:
Done